tests/rules/avoid-dispatching-multiple-actions-sequentially.test.ts

```diff
@@ -0,0 +1,78 @@
+import rule, {
+  ruleName,
+  messageId,
+} from '../../src/rules/avoid-dispatching-multiple-actions-sequentially'
+import { ruleTester } from '../utils'
+
+ruleTester().run(ruleName, rule, {
+  valid: [
+    `
+@Component()
+export class FixtureComponent {
+  name = 'Tim'
+  constructor(private store: Store<{}>){}
+
+  ping() {
+    this.store.dispatch({ type: 'PING' })
+  }
+}`,
+  ],
+  invalid: [
```

Comment:
Could you create each invalid block as a separate snippet?
That way, it's a bit easier to read imho.

Reply:
Will do !